### ISSUE TITLE
Feature Request - more Validators

### ISSUE THREAD

**Issue by niklasgrewe** [noahsalvi/svelte-use-form#7]
At the moment, the following validators are available:

* required
* minLength(n)
* maxLength(n)
* number
* email

It would be useful, if we have some more validators like for URL, Phone Number, Credit Card etc. 
I found this library: https://github.com/validatorjs/validator.js/

It is possible to integrate them into svelte-use-form?

**Comment by noahsalvi** [comment]
Yeah, adding more basic validators like URL is a good idea but for the more specific cases, I think that instead of wrapping every validator from validatorjs and adding it as a dependency, it would be more effective if the user just converts it themselves. That way the consumer has the responsibility if they want to use validatorjs or even something different.
https://svelte.dev/repl/21fc7637645d4917994ad4140b54b871?version=3.35.0

**Comment by niklasgrewe** [comment]
thanks for your REPL, yeah you're right. For more specific cases, its more useful to give the user the choice, which validator he wants to use, but it would be nice if some basic validator would still be added like URL 👍
